PWGHF/TableProducer/treeCreatorD0ToKPi.cxx

```diff
@@ -32,6 +32,17 @@ namespace o2::aod
 {
 namespace full
 {
+
+enum SelectionType {
+  SelD0 = BIT(0),
+  SelD0bar = BIT(1),
+  IsSignal = BIT(2),
+  ReflectedD0 = BIT(3),
+  ReflectedD0bar = BIT(4),
+  Prompt = BIT(5),
+  NonPrompt = BIT(6)
+};
```

Comment:
You don't need that many options, because reflection implies signal and prompt implies not non-prompt.
Similar kind of bitmap is implemented in https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/HFC/TableProducer/correlatorDMesonPairs.cxx#L47-L51
In your case, you should need just 5 bits: selD0, selD0bar, trueD0, trueD0bar, prompt.